subsys/bluetooth/controller/Kconfig

```diff
@@ -50,3 +49,119 @@ config BLECTLR_THREADSAFETY_LOCK_TIMEOUT_MSEC
 		locked.
 
 endmenu
+
+if FLASH
+config SOC_FLASH_NRF_LL_NRFXLIB
+	bool "Nordic BLE controller flash driver"
+	default y
+	select FLASH_HAS_PAGE_LAYOUT
+	select FLASH_HAS_DRIVER_ENABLED
+	select MPU_ALLOW_FLASH_WRITE if CPU_HAS_MPU
+	help
+		Enables Nordic BLE controller flash driver.
+
+endif # FLASH
+
+if ENTROPY_GENERATOR
+
+config ENTROPY_NRF_LL_NRFXLIB
+  bool "Nordic BLE controller entropy driver"
+	default y
+	depends on ENTROPY_GENERATOR
+	depends on SOC_COMPATIBLE_NRF
+	select ENTROPY_HAS_DRIVER
+	help
+	  This option enables the RNG peripheral, which is a random number
+	  generator, based on internal thermal noise, that provides a
+	  random 8-bit value to the host when read.
+
+endif # ENTROPY_GENERATOR
+
+config CLOCK_CONTROL_NRF_LL_NRFXLIB
+  bool "Nordic BLE controller clock control driver"
+	default y
+
+config CLOCK_CONTROL_NRF5_M16SRC_DRV_NAME
+	string "NRF5 16MHz clock device name"
+	default "clk_m16src"
+
+config CLOCK_CONTROL_NRF5_K32SRC_DRV_NAME
+	string "NRF5 32KHz clock device name"
+	default "clk_k32src"
+
+choice CLOCK_CONTROL_NR5_SOURCE
+	prompt "32KHz clock source"
+	default CLOCK_CONTROL_NRF5_K32SRC_XTAL
+
+config CLOCK_CONTROL_NRF5_K32SRC_RC
+	bool "RC Oscillator"
+
+config CLOCK_CONTROL_NRF5_K32SRC_XTAL
+	bool "Crystal Oscillator"
+
+config CLOCK_CONTROL_NRF5_K32SRC_SYNTH
+	bool "Synthesized from HFCLK"
+
+endchoice
+
+config CLOCK_CONTROL_NRF5_K32SRC_BLOCKING
+	bool "Blocking 32KHz crystal oscillator startup"
+	depends on CLOCK_CONTROL_NRF5_K32SRC_XTAL
+	help
+	  Clock control driver will spin wait in CPU sleep until 32KHz
+	  crystal oscillator starts up. If not enabled, RC oscillator will
+	  initially start running and automatically switch to crystal when
+	  ready.
+
+choice CLOCK_CONTROL_NR5_ACCURACY
+	prompt "32KHz clock accuracy"
+	default CLOCK_CONTROL_NRF5_K32SRC_500PPM if CLOCK_CONTROL_NRF5_K32SRC_RC
+	default CLOCK_CONTROL_NRF5_K32SRC_20PPM
+
+config CLOCK_CONTROL_NRF5_K32SRC_500PPM
+	bool "251 ppm to 500 ppm"
+
+config CLOCK_CONTROL_NRF5_K32SRC_250PPM
+	bool "151 ppm to 250 ppm"
+
+config CLOCK_CONTROL_NRF5_K32SRC_150PPM
+	bool "101 ppm to 150 ppm"
+
+config CLOCK_CONTROL_NRF5_K32SRC_100PPM
+	bool "76 ppm to 100 ppm"
+
+config CLOCK_CONTROL_NRF5_K32SRC_75PPM
+	bool "51 ppm to 75 ppm"
+
+config CLOCK_CONTROL_NRF5_K32SRC_50PPM
+	bool "31 ppm to 50 ppm"
+
+config CLOCK_CONTROL_NRF5_K32SRC_30PPM
+	bool "21 ppm to 30 ppm"
+
+config CLOCK_CONTROL_NRF5_K32SRC_20PPM
+	bool "0 ppm to 20 ppm"
+
+config CLOCK_CONTROL_NRF5_K32SRC_10PPM
+	bool "6 ppm to 10 ppm"
+
+config CLOCK_CONTROL_NRF5_K32SRC_5PPM
+	bool "3 ppm to 5 ppm"
+
+config CLOCK_CONTROL_NRF5_K32SRC_2PPM
+	bool "2 ppm"
+
+config CLOCK_CONTROL_NRF5_K32SRC_1PPM
+	bool "1 ppm"
+
+endchoice
+
+
+config ENTROPY_NRF5_PRI
```

Comment:
Do we need this?

Reply:
Yes, the driver is dependent on this symbol and it's not defined in the case when the NRFXLIB LL is selected `zephyr/drivers/entropy/Kconfig` guards against including it.